client/blocks/reader-full-post/_content.scss

```diff
@@ -27,13 +38,16 @@
 	}
 
 	h4 {
-		font-size: 18px;
-		font-weight: 700;
+		font-size: 32px;
 		margin: 0 0 8px;
 	}
 
 	h5 {
-		font-weight: 700;
+		font-size: 24px;
+	}
+
+	h6 {
+		font-size: 21px;
```

Comment:
This is close to but does not match: 

https://github.com/Automattic/wp-calypso/blob/a83f9acf2b30fc352da61669616f280df655134b/packages/typography/styles/variables.scss#L41